frontend/isoko/src/features/business/BusinessAPI.ts

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+import { environment } from '../../environment/environment';
+import { Business } from '../../types/GlobalTypes';
+
+export const fetchBusiness = async (businessId: string): Promise<Business> => {
+   const response = await axios.get(
+      `${environment.prodURL}/business/${businessId}`
+   );
+
+   const business = response.data.results;
```

Comment:
The business endpoint now returns the business object in just the body so you only need response.data here